front/src/pages/MainPage/components/Grid/components/Card/index.js

```diff
@@ -9,15 +9,15 @@ import Typography from "@material-ui/core/Typography";
 
 import { useStyles } from "./styles";
 
-const Card = ({ picture, title, author, pages, handleClick }) => {
+const Card = ({ book: { cover_url, title, author, pages }, handleClick }) => {
```

Comment:
tutaj lepiej zostawić book, jak wlozysz TS to zobaczysz ze wygodniej :)

Reply:
True ;).
Done.